src/analysis/inner_product_space/pi_L2.lean

```diff
@@ -98,6 +98,14 @@ lemma euclidean_space.nnnorm_eq {𝕜 : Type*} [is_R_or_C 𝕜] {n : Type*} [fin
   (x : euclidean_space 𝕜 n) : ∥x∥₊ = nnreal.sqrt (∑ i, ∥x i∥₊ ^ 2) :=
 pi_Lp.nnnorm_eq_of_L2 x
 
+lemma euclidean_space.dist_eq {𝕜 : Type*} [is_R_or_C 𝕜] {n : Type*} [fintype n]
+  (x y : euclidean_space 𝕜 n) : dist x y = (∑ i, dist (x i) (y i) ^ 2).sqrt :=
+by convert pi_Lp.dist_eq_of_L2 x y
```

Comment:
What's the `convert` doing here?

Reply:
It times out without and is instantaneous with it.

Reply:
Does `(pi_Lp.dist_eq_of_L2 x y : _)` work instead?

Reply:
It does. Real witchcraft.